msteams-platform/concepts/device-capabilities/media-capabilities.md

```diff
@@ -137,6 +137,77 @@ microsoftTeams.media.selectMedia(mediaInput, (error: microsoftTeams.SdkError, at
 });
 ```
 
+* Call `selectMedia` API for capturing videos using camera:
+
+  * Capturing videos with `fullscreen: true`:
+
+       `fullscreen: true` opens the camera in video recording mode. It provides an option to use both front and rear camera and also provides other attributes as mentioned in the following example:
+
+       ```javascript
+        
+         const defaultLensVideoProps: microsoftTeams.media.VideoProps = {
+             sources: [microsoftTeams.media.Source.Camera, microsoftTeams.media.Source.Gallery],
+             startMode: microsoftTeams.media.CameraStartMode.Photo,
+             cameraSwitcher: true,
+             maxDuration: 30
+        }
+         const defaultLensVideoMediaInput: microsoftTeams.media.MediaInputs = {
+             mediaType: microsoftTeams.media.MediaType.Video,
+             maxMediaCount: 6,
+             videoProps: defaultLensVideoProps
+        }
+       ```
+
+  * Capturing videos with `fullscreen: false`:
+
+       `fullscreen: false` opens the camera in video recording mode and uses the front camera only. Typically `fullscreen: false` is used when user wants to record video while reading content on the device screen.
+
+       This mode also supports `isStopButtonVisible: true`. This adds a stop button on the screen that allows user to stop the recording. If `isStopButtonVisible: false`, then it's expected that the recording is stopped by accepting an external selection to API.
```

Comment:
When isStopButtonVisible =  false, recording can be stopped either by calling mediaController API or when the recording duration reached to  maxDuration time specified in video props. Also we should be adding a section for mediaController API(the api called to stop the video programatically.)